sentry-ruby/lib/sentry/envelope/item.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module Sentry
+  # @api private
+  class Envelope::Item
+    STACKTRACE_FRAME_LIMIT_ON_OVERSIZED_PAYLOAD = 500
+    MAX_SERIALIZED_PAYLOAD_SIZE = 1024 * 1000
+
+    SIZE_LIMITS = Hash.new(MAX_SERIALIZED_PAYLOAD_SIZE).update(
+      "profile" => 1024 * 1000 * 10
```

Comment:
@sl0thentr0py wasn't sure what it should be exactly, so please let me know what value we want here and I'll adjust it + update the spec accordingly